audit/auditLog.json

```diff
@@ -58,39 +65,73 @@
     }
   },
   "auditedContracts": {
+    "EmergencyPauseFacet": {
```

Comment:
this PR changes the order of the audit log for no obvious reason (it was meant to be alphabetical)